test/jdk/java/awt/Mixing/AWT_Mixing/GlassPaneOverlappingTestBase.java

```diff
@@ -110,8 +125,53 @@ protected boolean performTest() {
         if (!super.performTest()) {
             return false;
         }
+        final CountDownLatch latch = new CountDownLatch(1);
+        f.addFocusListener(new FocusAdapter() {
+            @Override public void focusGained(FocusEvent e) {
+                latch.countDown();
+            }
+        });
+        if (testResize) {
+            wasLWClicked = false;
+            try {
+                SwingUtilities.invokeAndWait(new Runnable() {
+
+                    public void run() {
+                        testedComponent.setBounds(0, 0,
+                                testedComponent.getPreferredSize().width,
+                                testedComponent.getPreferredSize().height + 20);
+                        Component focusOwner = KeyboardFocusManager.getCurrentKeyboardFocusManager()
+                                .getFocusOwner();
+                        if (focusOwner == f) {
+                            // frame already had focus
+                            latch.countDown();
+                        } else {
+                            f.requestFocusInWindow();
+                        }
+                    }
+                });
+            } catch (InterruptedException ex) {
+                fail(ex.getMessage());
+            } catch (InvocationTargetException ex) {
+                fail(ex.getMessage());
+            }
```

Comment:
```suggestion
            } catch (InterruptedException | InvocationTargetException ex) {
                fail(ex.getMessage());
            }
```
Merge them together.